src/__tests__/index-test.js

```diff
@@ -16,6 +17,58 @@ import renderer from "react-test-renderer";
       expect(tree).toMatchSnapshot();
     });
 
+    test("emoji as children", () => {
```

Comment:
The tests are very on point 👌🏻 

I wonder what would happen if we put some emoji inside a nested HTML element on inside a nested React component. We should probably add a test case for each of these.

Reply:
Right now it only recognizes emojis that are direct children (as every non-string element is simply ignored). I think it would be quite complicated to add this functionality for nested emojis.

Reply:
Understood. I would still add some test cases to reflect this design decision, do you mind ?

Reply:
Done.